packages/mongo/mongo_driver.js

```diff
@@ -1037,6 +1049,28 @@ _.extend(SynchronousCursor.prototype, {
     }
   },
 
+  _nextObject: function () {
+    var self = this;
+    return self._nextObjectPromise().await();
+  },
+
+  // Returns the next object (like with _nextObject) or throws if it doesn't
+  // return within timeoutMS ms.
+  _nextObjectWithTimeout: function (timeoutMS) {
+    const self = this;
+    if (!timeout) {
```

Comment:
Should this be `if (!timeoutMs) {`? The test output doesn't give much information about why tests are failing, but this could be it.